doc/design/upgrade.md

```diff
@@ -26,7 +45,9 @@ There will be impact on container interaction while the appliance is down:
 - no attach ability
 - network outage for NAT based port forwarding
 
-All of those facets should resume normal operation after upgrade is complete
+All of those facets should resume normal operation after upgrade is complete. 
```

Comment:
@hickeng the full port mapping spec is stored in the config (parsable by: https://github.com/vmware/vic/blob/master/vendor/github.com/docker/go-connections/nat/nat.go#L128)